src-logger/bear/ResourceInvokerAdapter.php

```
@@ -0,0 +1,94 @@
+        $context = new BearSchemaLogContext(
+            BearSchemaLogContext::RESOURCE_REQUEST,
+            [
+                'uri' => (string) $request->resourceObject->uri,
+                'method' => $request->method,
+                'query' => $request->query,
+                'type' => 'resource_request',
+            ]
+        );
+        $this->schemaLogger->open($context);
+        
+        $startTime = microtime(true);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$startTime'. (Unused Code Rules)

(UnusedLocalVariable)